cas/cas.go

```diff
@@ -238,14 +245,48 @@ func (ds Store) WriteACI(r io.Reader) (string, error) {
 			BlobKey:    key,
 			AppName:    im.Name.String(),
 			ImportTime: time.Now(),
+			Latest:     latest,
 		}
 		return WriteACIInfo(tx, aciinfo)
 	}); err != nil {
 		return "", fmt.Errorf("error writing ACI Info: %v", err)
 	}
+
+	// The treestore uses acirenderer that needs cas.ResolveKey that will
+	// need the aciinfo already written in the db.
```

Comment:
Uh. Yes that statement is really unclear! The intention was to explain, that, if we really want to render the aci in the treestore inside WriteACI we have to do this after writing the ACIInfo but this will create problems with error handling:
* The ACI is correctly written when the aciinfo is saved. A successive operation done inside WriteACI like rendering in the treestore shouldn't return an error if it fails as the real purpose of WriteACI has been met.

Peraphs, given that this shouldn't be done per the successive comment, I should simply remove that statement...